itou/users/models.py

```diff
@@ -176,6 +177,19 @@ def save(self, *args, **kwargs):
         self.validate_unique()
         super().save(*args, **kwargs)
 
+    def can_edit_email(self, user):
+        if self == user:
+            return True
+
+        is_account_creator = bool(user.created_by and user.created_by == self)
+        if user.has_verified_email or not user.is_handled_by_proxy or not is_account_creator:
```

Comment:
Je pourrais nous économiser une requête mais ça rend le code un peu plus long. A votre avis ?

```python
    if user.is_handled_by_proxy and is_account_creator:
        if not user.has_verified_email:
            return True
    return False
```

Reply:
perso je préfère sur une ligne. D’ailleurs, tu peux enlever le `return True`/`return False` avec
```py
return not user.has_verified_email and user.is_handled_by_proxy and is_account_creator
```

Reply:
Ah bah oui, merci !
@stephane ou @kemar, à votre avis ?

Reply:
Avec le _one-liner_ de @Keirua l'évaluation va _short-circuiter_ au premier `and` _falsy_.

Du coup, tu peux mettre les requêtes à la fin du _one-liner_ et conserver cette solution élégante.

Reply:
C'est nickel, merci ! :ok_hand: